Header header
string robot_id
bool ready
turtlesim/Pose position